XDS_PROTOCOL.md

```diff
@@ -191,11 +191,17 @@ drop can't be tolerated, traffic drop could have been avoided by providing a
 CDS/EDS update with both __X__ and __Y__, then the RDS update repointing from
 __X__ to __Y__ and then a CDS/EDS update dropping __X__.
 
-In general, to avoid traffic drop:
-* Sequencing should be make before break.
-* LDS and CDS updates should arrive before the respective RDS and EDS updates.
-* CDS/EDS resources corresponding to routes in LDS/RDS should be available at
-  update.
+In general, to avoid traffic drop, sequencing of updates should follow a
+`make before break` model, wherein
+* CDS updates (if any) must always be pushed first.
+* EDS updates (if any) must arrive after CDS updates for the respective clusters.
+* LDS updates must arrive after corresponding CDS/EDS updates.
+* RDS updates related to the newly added listeners must arrive in the end.
+* Stale CDS clusters and related EDS endpoints (ones no longer being
+  referenced) can then be removed.
+
+xDS updates can be pushed independently if no new clusters/routes/listeners
```

Comment:
My previous comment on this wasn't reflected in the text here; there are some nuances that should be communicated that were there, in particular around the lack of CDS/EDS warming and the safety of independent updates for new clusters/routes that have no relationship with existing ones.